src/EPR.Calculator.Frontend/Views/CalculationRunDetails/Index.cshtml

```diff
@@ -21,10 +24,10 @@
                     </a>
                 </div>
                 <h2 class="gem-c-attachment__title govuk-body">
-                    <a href="@Url.Action("Index", "DownloadUnavailableError")" class="govuk-link" target="_self">Download</a><br />
+                    <a href="javascript:void(0)" onclick="downloadFile('@DownloadAPI', '@ErrorPage', event);" class="govuk-link" target="_self">Download</a><br />
```

Comment:
Please create Constants in C# Template and pass them instead of Javascript strings for '@DownloadAPI', '@ErrorPage',